tests/components/prometheus/test_init.py

```diff
@@ -1788,6 +1803,26 @@ async def switch_fixture(
     return data
 
 
+@pytest.fixture(name="fan_entities")
+async def fan_fixture(
+    hass: HomeAssistant, entity_registry: er.EntityRegistry
+) -> dict[str, er.RegistryEntry]:
+    """Simulate fan entities."""
+    data = {}
+    fan_1 = entity_registry.async_get_or_create(
+        domain=fan.DOMAIN,
+        platform="test",
+        unique_id="fan_1",
+        suggested_object_id="fan_1",
+        original_name="Fan 1",
+    )
+    set_state_with_entry(hass, fan_1, STATE_LOCKED)
```

Comment:
Is `STATE_LOCKED` one of the expected state values for `fan`? I'd expect it to be `STATE_ON` or `STATE_OFF` given that fan is a `ToggleEntity`:
https://github.com/home-assistant/core/blob/f09063d70636ae0f87379ebf78de31272fff2974/homeassistant/components/fan/__init__.py#L220

Reply:
that was copy pasta, fixed now